testutil/singlevm/cleanup.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 
-if [ ! -z $1 ]; then
-    hosts_file_backup=$1
-else
-    . ~/local/demo.sh
-    hosts_file_backup=$HOSTS_FILE_BACKUP
-fi
+. ~/local/demo.sh
```

Comment:
Is this necessary? I don't thing anything in this script actually depends on any of the env variables in the that file any longer.